sphinxcontrib/inmanta/api.py

```diff
@@ -16,8 +16,10 @@
     Contact: code@inmanta.com
 """
 
+import logging
+import typing
 from collections import defaultdict, OrderedDict
-from typing import List, Optional, Sequence, Tuple, Callable
+from typing import List, Optional, Sequence, Tuple, Callable, Mapping, Any, Union
```

Comment:
Most of these should be imported from `collections.abc` now. I especially think that we should import new imports from there.

Reply:
Also, `Any` doesn't seem to be used?